lib/rbi/rewriters/sort_nodes.rb

```diff
@@ -13,28 +13,28 @@ def visit(node)
         return unless node.is_a?(Tree)
 
         visit_all(node.nodes)
-        original_order = node.nodes.map.with_index.to_h
+        original_order = node.nodes.each_with_index.to_h
 
         # The child nodes could contain private/protected markers. If so, they should not be moved in the file.
         # Otherwise, some methods could see their privacy change. To avoid that problem, divide the array of child
         # nodes into chunks based on whether any Visibility nodes appear, and sort the chunks independently. This
         # applies the ordering rules from the node_rank method as much as possible, while preserving visibility.
-        sorted_nodes = node.nodes.chunk do |n|
-          n.is_a?(Visibility)
-        end.flat_map do |_, nodes|
```

Comment:
There's no Rubocop rule that can enforce this (last I checked), but having `end.something_else do ||` on the same line is kinda weird, and reads better when it's one operation per line IMO.